src/app/containers/FrontPageSection/index.jsx

```diff
@@ -70,43 +71,233 @@ const MarginWrapper = ({ firstSection, oneItem, children }) => {
   return children;
 };
 
-const StoryPromoComponent = ({ item, sectionNumber, storyNumber }) => {
-  const topStory = sectionNumber === 0 && storyNumber === 0;
+const StoryPromoComponent = ({ item, displayImage, leading, topStory }) => {
   const lazyLoadImage = !topStory; // don't lazy load image if it is a top story
 
   return (
-    <StoryPromo item={item} topStory={topStory} lazyLoadImage={lazyLoadImage} />
+    <StoryPromo
+      item={item}
+      topStory={topStory}
+      lazyLoadImage={lazyLoadImage}
+      displayImage={displayImage}
+      leading={leading}
+    />
   );
 };
+StoryPromoComponent.defaultProps = {
+  displayImage: true,
+  leading: false,
+  topStory: false,
+};
 
 StoryPromoComponent.propTypes = {
   item: shape(storyItem).isRequired,
+  displayImage: bool,
+  leading: bool,
+  topStory: bool,
+};
+
+const columns = {
+  normal: {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
```

Comment:
This first `3` groups can be stored on a `variable` and be reused as they don't change anywhere on the page. The only changes are on `group4` and `group5`